### ISSUE TITLE
Consider (discuss/mockup) example images to help people choose the right filter

### ISSUE THREAD

**Issue by jywarren** [publiclab/infragram#226]
This page shows examples of the two filter types (red or blue): https://publiclab.org/wiki/infragram#Conversion+types

But maybe it's helpful to have those example images on the presets modal itself:

![Screen Shot 2022-03-15 at 11 42 45 AM](https://user-images.githubusercontent.com/24359/158415823-4c9ae0b0-0ad0-4ee0-9fa9-7048f390034a.png)

The two images are:

Red filter (more common, more recent):

![Screen Shot 2022-03-15 at 11 41 07 AM](https://user-images.githubusercontent.com/24359/158415920-49aedb8f-2e48-4eac-b17d-759252ed2371.png)

Blue filter:

![Screen Shot 2022-03-15 at 11 40 58 AM](https://user-images.githubusercontent.com/24359/158415950-8c0e063a-f9e1-4ff8-8541-38a6292bcf1e.png)

Counter-intuitively, adding the red filter and [adjusting the white balance properly](https://publiclab.org/wiki/infragram#White+balance) results in a blue-ish image, and a blue filter results in a reddish image:

**Comment by jywarren** [comment]
So maybe this image could be useful?

![Screen Shot 2022-03-15 at 12 00 05 PM](https://user-images.githubusercontent.com/24359/158419721-e71382ef-db37-4be9-b640-4f8b1b459696.png)

**Comment by jywarren** [comment]
I think we should do some sketches/mockups and discuss here to figure out what would be the clearest way to present this!

**Comment by Ash-KODES** [comment]
Hello @jywarren,
I have a suggestion. We can provide two images for each filter in their respective rows. One image would be same for all the filters in each row(which would be the original image), and other image would be the filtered version of the original, based on the filter row in which it is shown.

So if it is blue filter row of the presets modal, then two images would be shown in it's row, one would be original image and the other would be blue filtered version of the original image and it will be similar for the red filter row, i.e. one image would be original image and one would be filtered version of it using the red filter.

We will just have to add two more columns to show the two images in each row showing the original image and and filtered image. I think that should suffice the purspose of explaining what each filter would do to the original image.

**Comment by PythonCoderUnicorn** [comment]
Hello Public Labs !

This is my first issue on this repo, I am new and from the Outreachy program.

Using your image uploader and step 2 analytics section, I tried 2 images and this image is the second one (as the first one did not render in any scale option). The HSV function does not work either from the grayscale or original color format. The slider above save has no effect. 

![Screen Shot 2022-03-25 at 4 39 48 PM](https://user-images.githubusercontent.com/55933131/160211227-57f33bb4-2ae3-41ae-8eaa-232d3305a59e.png)

**Comment by jywarren** [comment]
> Hello @jywarren, I have a suggestion. We can provide two images for each filter in their respective rows. One image would be same for all the filters in each row(which would be the original image), and other image would be the filtered version of the original, based on the filter row in which it is shown.
> 
> So if it is blue filter row of the presets modal, then two images would be shown in it's row, one would be original image and the other would be blue filtered version of the original image and it will be similar for the red filter row, i.e. one image would be original image and one would be filtered version of it using the red filter.
> 
> We will just have to add two more columns to show the two images in each row showing the original image and and filtered image. I think that should suffice the purspose of explaining what each filter would do to the original image.

Hi @Ash-KODES i'm interested... but it's a little hard to follow, do you think you'd be able to try doing a mockup, even using Google Presentions to show what you mean?

**Comment by jywarren** [comment]
> Using your image uploader and step 2 analytics section, I tried 2 images and this image is the second one (as the first one did not render in any scale option). The HSV function does not work either from the grayscale or original color format. The slider above save has no effect.

Hi @PythonCoderUnicorn welcome!!! Thank you -- I filed a bug for the HSV issue here: https://github.com/publiclab/infragram/issues/261

I also made this issue to try to address the slider problem: https://github.com/publiclab/infragram/issues/259

Thank you for highlighting these... if you find more, would you mind making new issues for them?

**Comment by Forchapeatl** [comment]
> Although this page points out that once you white balance the images look different:
> 
>     * https://publiclab.org/wiki/infragram#White+balance
> 
>     * https://publiclab.org/wiki/infrablue-white-balance
> 
> 
> So, I made a new diagram, showing the before and after white balancing versions:
> 
> ![Screen Shot 2022-03-15 at 12 24 43 PM](https://user-images.githubusercontent.com/24359/158425134-99e2f29d-268c-4995-bc13-da2f07cda293.png)

@jwarren , could we include a helper icon , which displays an image (before and after white balancing versions) tooltip on hover as shown  on the gif bellow. 
![presets](https://user-images.githubusercontent.com/24577149/162645685-7e6275e5-f625-411c-b7c0-395aa6615c85.gif)

**Comment by jywarren** [comment]
Oh, this is interesting! Thank you! What if instead of on hover, we used a
details element?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details
Thank you!!

**Comment by Forchapeatl** [comment]
> 



![presets2](https://user-images.githubusercontent.com/24577149/163059631-0aa64038-7b9d-46b6-a30e-c5049976f1af.gif)
